c/driver/postgresql/connection.cc

```diff
@@ -365,6 +370,45 @@ class PqGetObjectsHelper {
       params.push_back(std::string(table_name_));
     }
 
+    if (table_types_ != NULL) {
+      std::vector<std::string> table_type_filter;
+      const char** table_types = table_types_;
+      while (*table_types != NULL) {
+        auto table_type_str = std::string(*table_types);
+        if (auto search = PqGetObjectsHelper::PgTableTypes.find(table_type_str);
+            search != PqGetObjectsHelper::PgTableTypes.end()) {
+          table_type_filter.push_back(search->second);
+        }
+        table_types++;
+      }
+
+      if (!table_type_filter.empty()) {
+        std::ostringstream oss;
+        bool first = true;
+        oss << "(";
+        for (const auto& str : table_type_filter) {
+          if (!first) {
+            oss << ", ";
+          }
+          oss << "'" << str << "'";
+          first = false;
+        }
+        oss << ")";
+
+        if (StringBuilderAppend(&query, "%s%s", " AND c.relkind IN ",
```

Comment:
This does not use a parameter/prepared statement. My reasoning is that it is internally controlled which arguments end up being in the filter, and it was simpler to attach literally instead of keeping track of / enumerating all possible parameters in the statement